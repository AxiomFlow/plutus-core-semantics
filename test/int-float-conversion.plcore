(module Foo (imported) (exported () ()) (local () ())
  (define f1 (builtin intToFloat 2))
)
