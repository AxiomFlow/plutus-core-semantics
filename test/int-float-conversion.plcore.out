<T> <k> . </k> <env> ( Foo . f1 ) |-> 2e+00 </env> <mod> Foo </mod> </T>
